### PR TITLE
Add Bits.keepAlive() calls to get methods

### PR DESCRIPTION
**Pull request by DanHeidinga**
This is based on the changes in JDK11.

issue: eclipse/openj9#3469

Signed-off-by: Dan Heidinga <daniel_heidinga@ca.ibm.com>

### CODE REVIEW

**Comment by DanHeidinga** [comment]
Jenkins test all zlinux jdk8

**Comment by DanHeidinga** [comment]
Jenkins test all zlinux jdk8

File: jdk/src/share/classes/java/nio/Direct-X-Buffer.java.template [hunk 1]
```diff
@@ -274,17 +294,55 @@ class Direct$Type$Buffer$RW$$BO$
...
                 throw new BufferUnderflowException();
+            try {
+#if[!byte]
+                if (order() != ByteOrder.nativeOrder())
+                    Bits.copyTo$Memtype$Array(ix(pos), dst,
+                                          (long)offset << $LG_BYTES_PER_VALUE$,
+                                          (long)length << $LG_BYTES_PER_VALUE$);
+                else
+#end[!byte]
+                    Bits.copyToArray(ix(pos), dst, arrayBaseOffset,
```
Comment (pshipton) [inline jdk/src/share/classes/java/nio/Direct-X-Buffer.java.template]: lost some code here

Comment (DanHeidinga) [reply jdk/src/share/classes/java/nio/Direct-X-Buffer.java.template]: Close - I duplicated some code here.  Not sure how I did it but I duplicated some of the methods

**Comment by DanHeidinga** [comment]
Jenkins test all zlinux jdk8

**Comment by pshipton** [comment]
The extended test failure is a known issue, unrelated to this change.

**Comment by DanHeidinga** [comment]
FYI @andrewcraik @vijaysun-omr for your awareness as this correctness fix may have an impact on perf.

**Pull request merged**